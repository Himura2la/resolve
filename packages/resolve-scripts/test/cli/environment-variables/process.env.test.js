import path from 'path'

import exec from '../../exec'

test('process.env.XXX works correctly', async () => {
  const json = await exec(
    `resolve-scripts dev --config=${path.resolve(
      __dirname,
      './resolve.test.env.process.env.config.json'
    )}`
  )

  expect(json).toMatchSnapshot()
})
