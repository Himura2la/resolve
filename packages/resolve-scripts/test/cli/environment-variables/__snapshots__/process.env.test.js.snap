// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`process.env.XXX works correctly 1`] = `
Object {
  "aggregates": "common/aggregates/index.js",
  "auth": Object {
    "strategies": "auth/index.js",
  },
  "bus": Object {
    "adapter": "resolve-bus-memory",
    "options": Object {},
  },
  "deployOptions": Object {
    "applicationName": "resolve-scripts",
    "build": true,
    "config": "resolve.config.json",
    "env": Object {
      "$ref/deployOptions/env/360a705dd147444df6a1f3bdc5a4511edda7c09673cbf2a95bc43c8adbfef540": "process.env.HOST",
      "$ref/deployOptions/env/85995c84d52282ab45fe1aa54294508468cda44f3edc2fa4a405d3512246f848": "process.env.STORAGE_ADAPTER_OPTIONS_DB_NAME",
      "$ref/deployOptions/env/c46469501b5f86bc90b1fee9055515a1b4332936503fcbbb357c2b34f9d75701": "process.env.PORT",
      "$ref/deployOptions/env/e88d0f2c0135c80eaa967d5d47afc480c71aed0c2e43765a7089a098c739694e": "process.env.STORAGE_ADAPTER",
    },
    "inspectHost": "127.0.0.1",
    "inspectPort": 9229,
    "mode": "development",
    "openBrowser": true,
    "start": true,
    "watch": true,
  },
  "distDir": "dist",
  "host": "$ref/deployOptions/env/360a705dd147444df6a1f3bdc5a4511edda7c09673cbf2a95bc43c8adbfef540",
  "index": "client/index.js",
  "jwtCookie": Object {
    "maxAge": 31536000000,
    "name": "jwt",
  },
  "port": "$ref/deployOptions/env/c46469501b5f86bc90b1fee9055515a1b4332936503fcbbb357c2b34f9d75701",
  "protocol": "http",
  "readModels": "common/read-models/index.js",
  "redux": Object {
    "middlewares": "client/middlewares/index.js",
    "reducers": "client/reducers/index.js",
    "store": "client/store/index.js",
  },
  "rootPath": "",
  "routes": "client/routes.js",
  "sagas": "common/sagas/index.js",
  "staticDir": "static",
  "staticPath": "",
  "storage": Object {
    "adapter": "$ref/deployOptions/env/e88d0f2c0135c80eaa967d5d47afc480c71aed0c2e43765a7089a098c739694e",
    "options": Object {
      "dbName": "$ref/deployOptions/env/85995c84d52282ab45fe1aa54294508468cda44f3edc2fa4a405d3512246f848",
    },
  },
  "subscribe": Object {
    "adapter": "resolve-redux/dist/subscribe_adapter",
    "options": Object {},
  },
  "viewModels": "common/view-models/index.js",
}
`;
